examples/flipper.py

```diff
@@ -120,21 +121,22 @@
     r_flipper_body.velocity = l_flipper_body.velocity = 0, 0
 
     ### Remove any balls outside
-    to_remove = []
-    for ball in balls:
-        if ball.body.position.get_distance((300, 300)) > 1000:
-            to_remove.append(ball)
+    to_remove = [
```

Comment:
imho the explicit for was easier to read than this multi-line list comprehension. Same applies to other places with the same refactor